examples/placeholder/linux/apps/app1/ciTests.json

```diff
@@ -0,0 +1,4 @@
+{
+    "Test": ["Test_TC_BINFO_2_3_Simulated"],
+    "collection": ["Test"]
```

Comment:
Do we need the "collection" bits, or can we just enumerate the props on the object?  The .js had a list because it didn't have a good way to enumerate things...

Reply:
We don't need it if you don't want it. The reason why it's here, is because initially I simply 1-to-1 mimicked the functionality of  tests.json.

Reply:
I think its duplicating info as well. @krypton36 can you also please confirm ?